core/templates/pages/story-editor-page/modal-templates/story-editor-unpublish-modal.component.ts

```diff
@@ -18,21 +18,41 @@
 
 import { Component } from '@angular/core';
 import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
+import { PlatformFeatureService } from 'services/platform-feature.service';
 
 @Component({
   selector: 'oppia-story-editor-unpublish-modal',
   templateUrl: './story-editor-unpublish-modal.component.html'
 })
 export class StoryEditorUnpublishModalComponent {
   constructor(
+     private platformFeatureService: PlatformFeatureService,
      private activeModal: NgbActiveModal
   ) {}
 
+  badContentReasonText: string = 'Bad content (no new explorations ' +
+    'will be added)';
+
+  splitChapterReasonText: string = 'Split Chapters' +
+    ' (requires new explorations to be added)';
+
+  selectedReasonText: string = this.badContentReasonText;
+  unpublishingReason: string = 'BAD_CONTENT';
```

Comment:
Do we want `BAD_CONTENT` to be the default value? Or would it be better to have a placeholder such as `<Please select a reason>`?

Reply:
@gp201 As per the mock I feel it is better to keep the default value (It is not mentioned in the PRD to keep a placeholder by default), and also there will be nothing to display in the warning container if there is no reason selected, which does not look good as UI.

Reply:
I will defer this decision to your mentor.